ProcessOptimizer/tests/test_optimizer.py

```diff
@@ -440,3 +447,84 @@ def flat_score(x):
     res_noise = opt_noise.get_result()
     _, [_, res_std_reset] = expected_minimum(res_noise, return_std=True)
     assert isclose(res_std_no_white, res_std_reset, rel_tol=0.001)
+
+
+@pytest.mark.fast_test
+def test_estimate_single_x():
+    regressor = GaussianProcessRegressor(noise=0, alpha=0)
+    opt = Optimizer(
+        [(-2.0, 2.0), (-3.0, 3.0)],
+        base_estimator=regressor,
+        n_initial_points=1,
+    )
+    opt.tell([1, 1], 2)
+    X = opt.estimate([1, 1])[0]
+    assert_almost_equal(X.Y.mean, 2)
+    assert_almost_equal(X[0].mean, 2)  # test that indexing works
+    assert_almost_equal(X.Y.std, 0)
+    assert_almost_equal(X.mean, 2)
+    assert_almost_equal(X[1], 2)  # test that indexing works
+    assert_almost_equal(X.std, 0)
+
+
+@pytest.mark.fast_test
+def test_estimate_multiple_x():
+    regressor = GaussianProcessRegressor(noise=0, alpha=0)
+    opt = Optimizer(
+        [(-2.0, 2.0), (-3.0, 3.0)],
+        base_estimator=regressor,
+        n_initial_points=3,
+    )
+    opt.tell([[1, 1], [0, 0], [1, -1]], [2, -1, 5])
```

Comment:
Another format suggestion here: It may be nicer if you explicitly declared the input to tell as `x` and `y` in two lines of code above, so you can call `opt.tell(x, y)` and `opt.estimate(x)`? This goes for the next test as well (line 497 specifically).